src/react/databrowser/objects/details/Metadata.jsx

```diff
@@ -26,21 +26,22 @@ const selectOptions = systemMetadata.map(o => ({
     label: o.header,
 })).concat([userMetadataOption]);
 
-const isUserMetadata = type => type === METADATA_USER_TYPE;
-const isSystemMetadata = type => type === METADATA_SYSTEM_TYPE;
+const isUserType = type => type === METADATA_USER_TYPE;
+const isSystemType = type => type === METADATA_SYSTEM_TYPE;
 
 const convertToAWSMetadata = (items: MetadataItems) => {
     const userMetadata = {};
     const systemMetadata = {};
-    for (let i = 0; i < items.length; i++) {
-        if (isEmptyItem(items[i])) {
+    for (const item of items) {
+        const { type, key, value } = item;
+        if (isEmptyItem(item)) {
             continue;
         }
-        if (isUserMetadata(items[i].type)) {
-            userMetadata[items[i].key] = items[i].value;
+        if (isUserType(type)) {
+            userMetadata[key] = value;
         }
```

Comment:
personally I would use else if here as it would seem to me that an item would be either user or system type and not both. Could just be me though.